.babelrc

```diff
@@ -1,6 +1,7 @@
 {
   "presets": [
-  	"es2015",
-  	"react"
-  	]
-}
\ No newline at end of file
+  	"latest",
+  	"react",
+    "babel-preset-stage-0"
```

Comment:
Minor point, and this existed before these changes, but there are some mixed spaces and tabs here.